### DISCUSSION TITLE
Sum fields across table [Apr 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1641] [Discord Threads]
> 🙋‍♂️ **lulubryan**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### lulubryan@discord You can use Lookup to link records between tables. But current version of NocoDB does not allow you to use Formula's over Lookup entries. We are working on it.  
#  
> 🙋‍♂️ **lulubryan**  
#### Thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959531308831371326)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
